Taskfile.yml

```diff
@@ -51,13 +51,6 @@ tasks:
     cmds:
       - godoc -http=:6060
 
-  build-coreboot:
```

Comment:
I suppose this job was already obsolete?

Reply:
Yeah, it was obsolete for a while